README.adoc

```diff
@@ -31,26 +31,69 @@ When developing you can use Quarkus hot-reload
 mvn compile quarkus:dev
 ----
 
+== Talking to the endpoint
+
 .NOTE
 The backend requires a valid `x-rh-identity` to be supplied on calls.
 For testing purposes you can take the one that is stored in
 link:src/test/resources/rhid.txt It has accountId = '1234' set.
 
 
-== List existing policies
+=== List existing policies
 
 [source,shell]
 ----
 curl  -Hx-rh-identity:`cat src/test/resources/rhid.txt` http://localhost:8080/api/custom-policies/v1.0/policies
 ----
 
+By default this uses paging with a page-size of 10.
+You can select different pages, page sizes and sorting with the following query parameters:
+
+|===
+|Parameter|Default|Comment
+
+|page     | 0     | Page numbers start at 0
+|pageSize | 10    |
+|sortColumn |  | Sort can happen by column names ("name",
```

Comment:
@josejulio Is(n't) default sorting by internal id ?

Reply:
mmmh, Not sure what postgres does by default, but it looks like that, yes.

Reply:
It looks like default is by id (primary key) column. Should we perhaps make the name the default sort argument (unless we get the 'last updated' column)?